### PR TITLE
Build release package

### PR DESCRIPTION
**Pull request by wenwei-dev**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -599,6 +599,7 @@ SET(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
 SET(CPACK_PACKAGE_VERSION_MAJOR "0")
 SET(CPACK_PACKAGE_VERSION_MINOR "1")
```
Comment (linas) [inline CMakeLists.txt]: we should probably sets version major to 1

**Comment by bgoertzel** [comment]
Hmmm... FWIW, by my intuition, I think Atomspace itself deserves a version 1.* , but OpenCog (as separate from Atomspace) still feels like it's in version 0.* territory...

**Comment by AmeBel** [comment]
atomspace if 5.0.3 https://github.com/opencog/atomspace/blob/master/opencog/atomspace/version.h

**Comment by linas** [comment]
Heh. There have been 5 major redesigns of how the atomspace works, now. So, yeah.

**Comment by wenwei-dev** [comment]
For simplicity, I'll use the version defined in version.h for atomspace, and cogutil.h for cogutils and leave opencog as it is now, which is 0.1.4. Sounds okay?

File: CMakeLists.txt [hunk 2]
```diff
@@ -644,7 +646,14 @@ SET(CPACK_SOURCE_IGNORE_FILES
 	"^${PROJECT_BINARY_DIR}/opencog/cogserver/server/cogserver$"
 	"^${PROJECT_BINARY_DIR}/examples/server/derivedserver$"
 	"^${PROJECT_BINARY_DIR}/examples/hopfield/hopfield$"
+	"^${PROJECT_SOURCE_DIR}/opencog/python/__init__.py$"
 )
+
+SET(CPACK_PACKAGING_INSTALL_PREFIX "/usr/local")
+SET(CPACK_DEBIAN_PACKAGE_DEPENDS
+    "atomspace (>= 0.1.4)"
```
Comment (linas) [inline CMakeLists.txt]: current atomspace version is 5.0.3 and we should request at least that much ...

**Pull request merged**